frontend/src/libs/components/button/button.tsx

```diff
@@ -1,25 +1,37 @@
+import { type IconName } from '#libs/types/icon-name.type.js';
+
+import { Icon } from '../icon/icon.js';
 import styles from './styles.module.scss';
 
 type Properties = {
-  label: string;
+  label?: string;
```

Comment:
```suggestion
  label: string;
```
Let's always have label. Add `isLabelVisuallyHidden` property and display label with `visually-hidden` class if this property is `true`